libs/payments/ui/src/lib/server/terms-and-privacy.tsx

```diff
@@ -28,28 +28,33 @@ function GenericTerms({
   l10n,
 }: GenericTermsProps) {
   return (
-    <div className="legal-blurb" role="group" aria-labelledby={titleId}>
-      <p className="legal-heading" id={titleId}>
+    <section
```

Comment:
I don't think this should be revised to a `section`, as it would create a nested `section` for each group of links.
<img width="553" alt="Screenshot 2023-09-14 at 11 24 26 AM" src="https://github.com/mozilla/fxa/assets/28129806/7840ca35-1781-4d16-90ea-5887adae5ce2">

Reply:
Agreed. I recommend changing the highlighted line (wrapping `section`) to an `aside` as it's related information.